File: wlr/OGUR/OGUR/Entities/Wall.cs
using SPX.Core;
using OGUR.Management;
using SPX.Entities;
using System;
using System.Runtime.Serialization;

namespace OGUR.Entities
{
    [Serializable()]
    public class Wall : Entity
    {
        public Wall(SerializationInfo info, StreamingContext context):base(info, context){}

        public Wall(Point2 location)
        {
            Initialize(location, SpriteType.WALL, OGUR.EntityType.WALL,ZDepth.Wall);
            _isBlocking = true;
        }
    }
}

File: proj/src/aigilas/entities/Wall.java
package aigilas.entities;

import aigilas.management.SpriteType;
import spx.bridge.DrawDepth;
import spx.bridge.EntityType;
import spx.core.Point2;
import spx.entities.Entity;

public class Wall extends Entity {
    public Wall(Point2 location) {
        Initialize(location, SpriteType.WALL, EntityType.WALL, DrawDepth.Wall);
        _isBlocking = true;
    }
}
